crates/oxc_ast/src/lib.rs

```diff
@@ -8,7 +8,6 @@
 //! * `"serde"` enables support for serde serialization
 
 #![feature(let_chains)]
-#![feature(is_some_and)]
```

Comment:
warning: the feature `is_some_and` has been stable since 1.71.0-nightly and no longer requires an attribute to enable